### PR TITLE
[videocenter-bp] Updates

### PR DESCRIPTION
**Pull request by jvega190**
Disable clicking on past events #3849 https://github.com/craftercms/craftercms/issues/3849
Allow countdown on upcomit events that are going live soon #3850 https://github.com/craftercms/craftercms/issues/3850
Added 'Active Events' query on Live events view

### CODE REVIEW

**Comment by jvega190** [comment]
@rart

File: sources/app/src/components/Cards/Cards.js [hunk 1]
```diff
@@ -205,4 +205,21 @@ class Cards extends Component {
     }
 }
 
+class CardContainer extends Component {
+  render() {
+    const { category, card, children } = this.props;
+    let videoName = card.title_s ? (card.title_s).toLowerCase().replace(/ /g, '-') : '';
+    videoName = encodeURI(videoName);
```
Comment (rart) [inline sources/app/src/components/Cards/Cards.js]: encodeURIComponent

File: sources/app/src/containers/Video/Video.js [hunk 2]
```diff
@@ -214,6 +214,15 @@ class Video extends Component {
         );
     }
 
+    autoLoadVideo() {
+      this.loadVideo(this.props);
+
+      this.setState({
+        ...this.state,
```
Comment (rart) [inline sources/app/src/containers/Video/Video.js]: setState already merges on it's own

File: sources/app/start.sh [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-REACT_APP_SITE_NAME=video-center REACT_APP_BASE_URL=http://localhost:8080 react-scripts start
+REACT_APP_SITE_NAME=vc-repo REACT_APP_BASE_URL=http://localhost:8080 react-scripts start
```
Comment (rart) [inline sources/app/start.sh]: This is too variable. This should be blank and the code should use the cookie if blank.

**Comment by jvega190** [comment]
@rart can you please check the updates?

**Pull request merged**